leshoya.rb

```diff
@@ -0,0 +1,176 @@
+# PRODUCT HAS A NAME, DIAMETER, MASS, MOONS, GRAVITY, AND DISTANCE FROM THE SUN VALUES
+class Planet #always camel case
+
+  attr_reader :name, :diameter, :mass, :moons, :gravity, :distance_from_the_sun
+  attr_writer :name, :diameter, :mass, :moons, :gravity, :distance_from_the_sun
```

Comment:
```rb
attr_accessor :name, :diameter, :mass, :moons, :gravity, :distance_from_the_sun
```

:point_up: produces the same results as lines 4 and 5. `attr_accessor` is a combination of both `attr_reader` and `attr_writer`.